### ISSUE TITLE
hanami beta needs gemfile installed

### ISSUE THREAD

**Issue by dsisnero** [hanami/hanami#838]
bin\hanami in beta requires bundler and uses Bundle.require.  The gem doesn't include a Gemfile so the command fails on windows

c:\Dominic\programming\ruby\hanami\hanami-1.1.0.beta2\bin>hanami new bookshelf
Bundler::GemfileNotFound: Could not locate Gemfile or .bundle/ directory
dir

c:\Dominic\programming\ruby\hanami\hanami-1.1.0.beta2\bin>gem list hanami

*** LOCAL GEMS ***

hanami (1.1.0.beta2)
hanami-assets (1.1.0.beta2)
hanami-cli (0.1.0.beta2)
hanami-controller (1.1.0.beta2)
hanami-helpers (1.1.0.beta2)
hanami-mailer (1.1.0.beta2)
hanami-router (1.1.0.beta2)
hanami-utils (1.1.0.beta2)
hanami-validations (1.1.0.beta2)
hanami-view (1.1.0.beta2)

From an unpacked hanami -- gem unpack hanami

 Directory of c:\Dominic\programming\ruby\hanami\hanami-1.1.0.beta2

10/03/2017  04:59 PM    <DIR>          .
10/03/2017  04:59 PM    <DIR>          ..
10/03/2017  04:59 PM    <DIR>          bin
10/03/2017  04:59 PM            24,064 CHANGELOG.md
10/03/2017  04:59 PM             3,230 CODE_OF_CONDUCT.md
10/03/2017  04:59 PM             9,163 FEATURES.md
10/03/2017  04:59 PM             1,792 hanami.gemspec
10/03/2017  04:59 PM    <DIR>          lib
10/03/2017  04:59 PM             1,071 LICENSE.md
10/03/2017  04:59 PM             4,694 README.md
               6 File(s)         44,014 bytes
               4 Dir(s)   8,635,719,680 bytes free

**Comment by jodosha** [comment]
@dsisnero thanks for reporting it. I'm working on a fix.

**Comment by jodosha** [comment]
Another case reported by @malin-as in [chat](https://gitter.im/hanami/chat?at=59d479bcb59d55b8236e911f):

```shell
$ echo '2.4.2' > .ruby-version
$ echo 'hanami-beta2' > .ruby-gemset
$ cd .
ruby-2.4.2 - #gemset created /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2
ruby-2.4.2 - #generating hanami-beta2 wrappers..........
$ gem install hanami --pre
[...]
28 gems installed

$ hanami new bookshelf
/home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:212:in `rescue in root': Could not locate Gemfile or .bundle/ directory (Bundler::GemfileNotFound)
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:208:in `root'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:78:in `bundle_path'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:505:in `configure_gem_path'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:493:in `configure_gem_home_and_path'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:64:in `configure'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:128:in `definition'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:95:in `setup'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/bundler-1.15.4/lib/bundler.rb:108:in `require'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/gems/hanami-1.1.0.beta2/bin/hanami:5:in `<top (required)>'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/bin/hanami:23:in `load'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/bin/hanami:23:in `<main>'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/bin/ruby_executable_hooks:15:in `eval'
        from /home/malina/.rvm/gems/ruby-2.4.2@hanami-beta2/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by jodosha** [comment]
Fixed by #839
